server/service/src/sync/translations/invoice.rs

```diff
@@ -361,6 +363,12 @@ fn invoice_type(_type: &LegacyTransactType, name: &NameRow) -> Option<InvoiceRow
             _ => return None,
         };
     }
+    if name.code == REPACK_NAME_CODE {
```

Comment:
inventory adjustment is a bit of an edge case (i.e. the type in mSupply looks like normal invoice, but in omSupply it's a different invoice type). I think `sr` types can be matched in match statement below, rather then by name